mikenthiwa/jumia_replica_back
# #166281137 Set up circleCI

#### What does this PR do?
- Setting up circleCI for the project.

#### Description of Task to be completed?
- Ensure that the code is running in circleCI.

#### How should this be manually tested?
1. Git clone the repository.
2. Checkout to the `ch-set-up-circleCI-166281137` branch.
3. Run `yarn install`
4. Start the server `yarn start:dev` _(or the `npm` equivalent).
5. On your browser follow this link and see the jobs running in circleCI: `https://circleci.com/gh/mikenthiwa/workflows`

#### What are the relevant pivotal tracker stories?
[166281137](https://www.pivotaltracker.com/story/show/166281137)

#### Screenshots (If applicable)
![Screenshot 2019-05-29 at 09 35 50 (2)](https://user-images.githubusercontent.com/30254491/58534994-6b60fe00-81f5-11e9-9fe7-f979bb800cea.png)
